tests/BUILD

```diff
@@ -38,6 +38,7 @@ rust_test(
         "@crates//:serial_test",
         "@crates//:smol",
         "@crates//:tokio",
+        "@vaticle_typeql//rust:typeql_lang",
```

Comment:
A good ordering of dependencies would by "this" repository first, then "vaticle" dependencies, then "crate" dependencies!

Reply:
cc @dmitrii-ubskii  for you to take not of for yourself too ;) This is the level of detail I'm going to be pushing you toward over time

Reply:
That's the opposite of how we've been doing it so far jsyk.

Reply:
You can fix it at some point :) no rush atm

Reply:
Reordered